geetools/ee_dictionary.py

```diff
@@ -80,3 +82,116 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(
+        self, valueType: Literal["dict", "list", "value"] = "value"
+    ) -> ee.FeatureCollection:
+        """Convert a :py:class:`ee.Dictionary` to a :py:class:`ee.FeatureCollection` with no geometries (table).
+
+        There are 3 different type of values handled by this method:
+
+        1. value (default): when values are a `ee.String` or `ee.Number`, the
+           keys will be saved in the column "system:index" and the values in the
+           column "value".
+
+        2. dict: when values are a ee.Dictionary, the keys will be saved in the
+           column "system:index" and the values will be treated as each
+           Feature's properties.
+
+        3. list: when values are a ee.List of numbers or strings, the keys will
+           be saved in the column "system:index" and the values in as many
+           columns as items in the list. The column name pattern is "value_{i}"
+           where i is the position of the element in the list.
+
+        These are the only supported patterns. Other patterns should be converted
+        to one of these. For example, the values of a reduction using the
+        reducer `ee.Reducer.frequencyHistogram()` are of type `ee.Array` and
+        the array contains lists.
+
+        Examples:
```

Comment:
Can you move the Example section as the last item in the docstring ? 
Currently all the example will appear in the documentation before the parameters and the return statement.